app/webhook.go

```diff
@@ -789,8 +789,13 @@ func (a *App) HandleIncomingWebhook(c *request.Context, hookID string, req *mode
 		overrideIconURL = req.IconURL
 	}
 
-	_, err := a.CreateWebhookPost(c, hook.UserId, channel, text, overrideUsername, overrideIconURL, req.IconEmoji, req.Props, webhookType, "")
-	return err
+	if hook.Enabled {
```

Comment:
This is something we can check as soon as we have access to the struct/field. I'm thinking we should check it at that point, since it's a simple boolean check